content/sensu-go/5.21/installation/install-sensu.md

```diff
@@ -416,6 +416,57 @@ For a complete list of configuration options, see the [agent reference][7].
 Sensu keepalives are the heartbeat mechanism used to ensure that all registered agents are operating and can reach the Sensu backend.
 To confirm that the agent is registered with Sensu and is sending keepalive events, open the entity page in the [Sensu web UI][24] or run `sensuctl entity list`.
 
+### 4. Verify an example event
+
+With your backend and agent still running, send this request to the Sensu events API:
+
+{{< code shell >}}
+curl -X POST \
+-H 'Content-Type: application/json' \
+-d '{
+  "check": {
+    "metadata": {
+      "name": "check-mysql-status"
+    },
+    "status": 1,
+    "output": "could not connect to mysql"
+  }
+}' \
+http://127.0.0.1:3031/events
+{{< /code >}}
+
+This request creates a `warning` event that you can [view in your web UI Events page][32].
+
+To create an `OK` event, change the `status` to `0` and resend.
+You can change the `output` value to `connected to mysql` to see a different message for the `OK` event.
+
+## Next steps
+
+Now that you have installed Sensu, you’re ready to build your observability pipelines!
+Here are some ideas for next steps.
+
+### Do something new with Sensu
+
+If you're ready to see what Sensu can do, one of these pathways can get you started:
+
+- Manually trigger an event that [sends alerts to your email inbox][12].
+- [Create a check to monitor CPU usage][9] and [send Slack alerts based on your check][10].
+- [Collect metrics][33] with a Sensu check and use a handler to [populate metrics in InfluxDB][37].
+- Use the [sensuctl dump][38] command to export all of your events and resources as a backup &mdash; then use [sensuctl create][39] to restore if needed.
+
+### Deploy Sensu outside your local development environment
+
+If you're going to deploy Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).
+The next step in setting up TLS is to [generate the certificates you need][2].
+
```

Comment:
Need to add brief explanation of cluster and explain that users should decide at this point whether they want to run a cluster. This should not be separate from the "Deploy outside..." section.